tests/test_copy_novaseqx_flow_cell.py

```diff
@@ -0,0 +1,180 @@
+from pathlib import Path
+import pytest
+from typing import Set
+from cg.cli.demultiplex.copy_novaseqx_data import get_latest_analysis_directory
+
+from cg.cli.demultiplex.demux import (
+    copy_flow_cell_analysis_data,
+    is_ready_for_post_processing,
+)
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(name= "", scope= "")
```

Reply:
Renaming the fixture with the name parameter seems to be an anti-pattern so I will not change this. Using the default scope (per test) for simple string fixtures is fine since it does not take any time to create them